pages/common/pbmtopi3.md

```diff
@@ -0,0 +1,9 @@
+# pbmtopi3
+
+> Convert a PBM image into an Atari Degas PI3 image.
+> See also: `pi3topbm`.
+> More information: <https://netpbm.sourceforge.net/doc/pbmtopi3.html>.
+
+- Convert a PBM image into an Atari Degas PI3 image:
+
+`pbmtopi3 {{path/to/file.pbm}} > {{path/to/output.pi3}}`
```

Comment:
```suggestion
`pbmtopi3 {{path/to/image.pbm}} > {{path/to/atari_image.pi3}}`
```
Maybe we should use `output.pi3` for consistency with the other pages, but I prefer clarity over consistency in this specific case.

Not that the description is not clear, but this way one can directly look up the command example and have a good idea of what it does

Reply:
sure